UHPostalService/UHPostalService/Pages/Account/Employees/Index.cshtml.cs

```diff
@@ -124,6 +109,28 @@ public async Task OnGetAsync(string sortOrder, string searchString, int filterby
             {
                 EmployeeIdent = EmployeeIdent.Where(s => s.Id == flop);
             }
+            
+            switch (sortOrder)
+            {
+                case "name_desc":
+                    EmployeeIdent = EmployeeIdent.OrderByDescending(s => s.Name);
+                    break;
+                case "Date":
+                    EmployeeIdent = EmployeeIdent.OrderBy(s => s.Email);
+                    break;
+                case "Store":
+                    EmployeeIdent = EmployeeIdent.OrderBy(s => s.Store);
+                    break;
+                case "Role":
+                    EmployeeIdent = EmployeeIdent.OrderBy(s => s.Role);
```

Comment:
the store and role sorts dont have descending versions. Also sorting by just store isnt right you need to sort by store ID or something